runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -26,10 +26,6 @@ pub enum Cost {
     ActionDeleteKey,
     ActionDeleteAccount,
 
-    // TODO: Remove these two, compilation is a part of `ActionDeploy`.
-    ContractCompileBase,
-    ContractCompileBytes,
```

Comment:
I'd actually maybe kept these, as a "helper cost" to answer the question of "how much of deploy cost is compilation, and how much is everything else" . So, move it to the block below, and actually add the relevan functiosn to `lib.rs`